src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/Constants.cs

```diff
@@ -23,14 +24,37 @@ static Constants()
 
             Debug.Assert(assemblyVersion != null);
 
+
+            var majorVersion = typeof(Constants).Assembly.GetName().Version.Major;
+            var minorVersion = typeof(Constants).Assembly.GetName().Version.Minor;
+            var os = getOperatingSystem();
+            var runtime = ".NET";
+            var runtimeVersion = RuntimeInformation.FrameworkDescription;
+
             // assembly version attribute should always be present
             // but in case it isn't then don't include version in user-agent
```

Comment:
It looks like now instead of just skipping the assembly version, this now omits the entire user agent.

Reply:
Good catch